README.md

```diff
@@ -367,33 +367,48 @@ cd atoma-node
 
 The application uses a TOML configuration file with the following sections:
 
-##### `[atoma-service]`
+##### `[atoma_service]`
 
-- `chat_completions_service_url` (optional): Endpoint URL for the inference service. At least one of the service URLs must be provided.
-- `embeddings_service_url` (optional): Endpoint URL for the embeddings service. At least one of the service URLs must be provided.
-- `image_generations_service_url` (optional): Endpoint URL for the image generations service. At least one of the service URLs must be provided.
+- `chat_completions_service_urls`: Map of model names to endpoint URLs for the inference service (e.g., `{ "meta-llama/Llama-3.2-3B-Instruct" = "http://chat-completions:8000"}`)
```

Comment:
I believe this should still be optional

Reply:
Actually, this should be deserialized to a proper `HashMap`, so it can't be optional.